saq/__main__.py

```diff
@@ -26,7 +26,14 @@ def main():
     parser.add_argument(
         "--web",
         action="store_true",
-        help="Start web app",
+        help="Start web app. "
+        "By default, this only monitors the current worker's queue. To monitor multiple queues, see '--web-settings'",
+    )
+    parser.add_argument(
+        "--additional-web-settings",
```

Comment:
web-settings vs additional-web-settings as stated in 30

Reply:
what about extra_settings, or monitor_settings, or just web_settings is fine

Reply:
I think every word in "additional-web-settings" adds information:
settings: settings path
web: only for web
additional: in _addition_ to the main settings

But I'll settle for `web_settings` :)

Reply:
up to you, you can make the decision